plugins/examples/sidebar/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "sidebar",
+  "version": "0.0.1",
+  "description": "Your Headlamp plugin",
+  "scripts": {
+    "start": "headlamp-plugin start",
+    "build": "headlamp-plugin build",
+    "format": "headlamp-plugin format",
+    "lint": "headlamp-plugin lint",
+    "lint-fix": "headlamp-plugin lint --fix",
+    "tsc": "tsc"
+  },
+  "keywords": [
+    "headlamp",
+    "headlamp-plugin",
+    "kubernetes",
+    "plugins"
+  ],
+  "prettier": "@kinvolk/eslint-config/prettier-config",
+  "eslintConfig": {
+    "extends": [
+      "@kinvolk",
+      "prettier",
+      "plugin:jsx-a11y/recommended"
+    ]
+  },
+  "devDependencies": {
+    "@kinvolk/headlamp-plugin": "^0.4.3"
```

Comment:
`npm run tsc` inside the plugin isn't working for me. I think because we need to release a new headlamp-plugin which includes those typescript changes?